lib/generator._js

```diff
@@ -461,6 +454,9 @@ ScriptGenerator.prototype.generateAspNetCoreScript = function (templateFileName,
   argNotNull(templateFileName, 'templateFileName');
 
   var lowerCaseScriptType = this.scriptType.toLowerCase();
+  var fixedRestore = this.scriptType === ScriptType.batch ? fixPathSeperatorToWindows(options.restore) : fixPathSeperatorToUnix(options.restore);
```

Comment:
Still thinking what the right fix is. Are we able to assume that this script runs only on Kudu? I ask because in order to determine the path separator we need to know which platform the generated deployment script is expected to run. Batch and Bash are fine, but PoSH scripts run on both environments. So it's not possible to infer the correct path separator only from the script type.